pilot/pkg/security/model/authentication.go

```diff
@@ -76,12 +86,19 @@ func ConstructSdsSecretConfigForCredential(name string) *tls.SdsSecretConfig {
 		return nil
 	}
 
+	if features.EnableSDSServer && features.EnableXDSIdentityCheck {
+		return &tls.SdsSecretConfig{
+			Name:      "k8s://" + name,
```

Comment:
Why add the prefix?

Reply:
https://docs.google.com/document/d/1DEUkk0ocrD_oAH0h8TOUIC94Lf8msDXJb9lyNWlX_Ho